### PR TITLE
VY-RES-514 CRUD operation for OperationalResource

### PR DESCRIPTION
**Pull request by prkandel**
Crud operation for operationalResource

### CODE REVIEW

File: src/main/java/com/lftechnology/vyaguta/resource/rs/OperationalResourceRs.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+
+import com.lftechnology.vyaguta.commons.exception.ObjectNotFoundException;
+import com.lftechnology.vyaguta.commons.util.MultivaluedMapConverter;
+import com.lftechnology.vyaguta.resource.entity.OperationalResource;
+import com.lftechnology.vyaguta.resource.entity.ProjectRole;
+import com.lftechnology.vyaguta.resource.service.OperationalResourceService;
+
+@Path("/operationalresources")
```
Comment (lf-kailashraj) [inline src/main/java/com/lftechnology/vyaguta/resource/rs/OperationalResourceRs.java]: @lf-achyutpkl how about /resource/operational?

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

File: resources/database/vyaguta-resource.sql [hunk 2]
```diff
@@ -262,6 +262,19 @@ ALTER TABLE ONLY contract_member_histories
     ADD CONSTRAINT contract_member_histories_pk PRIMARY KEY (id);
 ALTER TABLE ONLY contract_member_histories 
 	ADD CONSTRAINT project_history_root_fk FOREIGN KEY(batch_id) REFERENCES project_history_root ON DELETE CASCADE;
+
+
+CREATE TABLE operational_resources (
+    id uuid NOT NULL,
+    employee_id uuid,
```
Comment (lf-achyutpkl) [inline resources/database/vyaguta-resource.sql]: how about adding not null and unique constraints??

File: src/main/java/com/lftechnology/vyaguta/resource/rs/OperationalResourceRs.java (hunk 1 above)
Comment (lf-achyutpkl) [inline src/main/java/com/lftechnology/vyaguta/resource/rs/OperationalResourceRs.java]: /resources/operational/ looks clean

File: src/main/java/com/lftechnology/vyaguta/resource/entity/OperationalResource.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+@Table(name = "operational_resources")
+@Entity
+public class OperationalResource extends BaseEntity implements Serializable {
+
+    private static final long serialVersionUID = 2173510009809546297L;
+    
+    @AttributeOverrides(@AttributeOverride(name = "id", column = @Column(name = "employee_id") ))
+    private Employee employee;
+
+    public UUID getEmployeeId() {
```
Comment (lf-achyutpkl) [inline src/main/java/com/lftechnology/vyaguta/resource/entity/OperationalResource.java]: @prkandel class has Employee property

**Pull request merged**